api/cds.proto

```diff
@@ -272,4 +272,28 @@ message Cluster {
     repeated LbSubsetSelector subset_selectors = 3;
   }
   LbSubsetConfig lb_subset_config = 22;
+  
+  message RingHashLbConfig {
+    // The hash function used to build the consistent hashing ring by hashing
+    // each server's address.
+    enum HashFunction {
+      // Standard library
+      STL = 0;
+      // XXHash
+      XXH64 = 1;
+    }
+    HashFunction hash_function = 1;
+
+    // Minimum hash ring size, i.e. total virtual nodes. A larger size
+    // will provide better request distribution since each host in the
+    // cluster will have more virtual nodes.
+    google.protobuf.UInt64Value minimum_ring_size = 2;
```

Comment:
Can you comment on default if not set and also runtime override capability.

Reply:
I was thinking of deleting the runtime override since it really doesn't make sense unless it's at a per-cluster level. In general it doesn't seem very useful beyond experimentation.

Reply:
That's fine I agree it isn't that useful.